eng/PatchConfig.props

```diff
@@ -20,6 +20,7 @@ Later on, this will be checked using this condition:
       Microsoft.AspNetCore.Server.IIS;
       Microsoft.AspNetCore.Server.IISIntegration;
       Microsoft.AspNetCore.Server.IntegrationTesting.IIS;
+      Microsoft.AspNetCore.Server.Kestrel.Core;
```

Comment:
No, your `release/2.2` branch is behind. Please rebase then add this to the new block that's for 2.2.2.